app/src/main/java/com/framgia/soundclound/screen/playtrack/PlayTrackViewModel.java

```diff
@@ -1,27 +1,77 @@
 package com.framgia.soundclound.screen.playtrack;
 
+import android.content.ComponentName;
+import android.content.ServiceConnection;
 import android.databinding.BaseObservable;
+import android.databinding.Bindable;
+import android.os.AsyncTask;
+import android.os.IBinder;
+import android.util.Log;
 import android.view.View;
+import android.widget.ImageView;
+import android.widget.SeekBar;
 
+import com.framgia.soundclound.R;
 import com.framgia.soundclound.data.model.Track;
+import com.framgia.soundclound.data.source.local.SharePreferences;
+import com.framgia.soundclound.service.MusicService;
+import com.framgia.soundclound.util.StringUtil;
 
 import java.util.List;
+import java.util.logging.Logger;
 
 /**
  * Created by Sony on 1/11/2018.
  */
 
-public class PlayTrackViewModel extends BaseObservable {
-    private List<Track> mTrackList;
+public class PlayTrackViewModel extends BaseObservable implements SeekBar.OnSeekBarChangeListener {
+    private String mTotalDuration;
+    private String mCurentDuration;
+    private int mProgressSeekBar = 0;
+    private List<Track> mTracks;
+    private int mIndexCurrentTrack;
     private Track mTrack;
+    private static MusicService mMusicService;
+    private boolean mMusicBound = false;
+    public OnClickTrackListener mOnClickTrackListener;
```

Comment:
WARNING: Variable 'mOnClickTrackListener' must be private and have accessor methods.